wpilibj/src/main/java/edu/wpi/first/wpilibj/romi/RomiGyro.java

```diff
@@ -114,11 +116,34 @@ public double getAngleZ() {
   }
 
   /** Reset the gyro angles to 0. */
+  @Override
   public void reset() {
     if (m_simAngleX != null) {
       m_angleXOffset = m_simAngleX.get();
       m_angleYOffset = m_simAngleY.get();
       m_angleZOffset = m_simAngleZ.get();
     }
   }
+
+  @Override
+  public void calibrate() {
+    // no-op - Romi Gyro calibration is done via web UI
```

Comment:
Maybe we should do the pluming to make this work from code too?

Reply:
We could definitely think about it. It will be a significant amount of plumbing to do though, since we'll have to add new WS messages (and appropriate handlers). The current calibration routine also takes about 25-30 seconds to run (just to make sure we get good offset values, since the IMU tends to be a little noisy)